README.rst

```diff
@@ -12,9 +12,40 @@
 .. image:: doc/source/_static/gnocchi-logo.png
 
 Gnocchi is an open-source, multi-tenant timeseries, metrics and resources
-database. It provides an `HTTP REST`_ interface to create and manipulate the
-data. It is designed to store metrics at a very large scale while providing
-access to metrics and resources information and history.
+database.
+
+The problem that Gnocchi solves is storage and indexing of time series data and
+resources at large scale, especiallly in the context of cloud computing. Modern
```

Comment:
s/at large scale/at a large scale/ (although i don't like the word large scale since it's ambiguous)

s/especiallly/especially/ i don't think it's the right word either but i can't think of the more appropriate word here. "including"?

Reply:
fixed